pkg/security/tests/span_test.go

```diff
@@ -0,0 +1,77 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build functionaltests
+
+package tests
+
+import (
+	"os"
+	"syscall"
+	"testing"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/security/model"
+	sprobe "github.com/DataDog/datadog-agent/pkg/security/probe"
+	"github.com/DataDog/datadog-agent/pkg/security/rules"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSpan(t *testing.T) {
+	ruleDef := &rules.RuleDefinition{
+		ID:         "test_span_rule",
+		Expression: `open.file.path == "{{.Root}}/test-span"`,
+	}
```

Comment:
Should we test exec events too ?

Reply:
It is not possible  :(, as it requires to control the thread-id

Reply:
👍🏻